Description: Fix logging invocation
 Compiling with hardening enabled broke due to this use of logging. With
 the patch applied, we no longer see the "no arguments with non-literal
 format string" error. The patch seems to me entirely safe. Please apply.
 .
 libgphoto2 (2.5.0-SprezzOS1) unreleased; urgency=low
 .
   * Update to 2.5.0.
Author: nick black <nick.black@sprezzatech.com>

---
Origin: SprezzOS, https://github.com/dankamongmen/sprezzos-world/commit/0ca55194c06466313aaf4daa014d3be3a4e4f058

--- libgphoto2-2.5.0.orig/libgphoto2_port/libgphoto2_port/gphoto2-port-info-list.c
+++ libgphoto2-2.5.0/libgphoto2_port/libgphoto2_port/gphoto2-port-info-list.c
@@ -306,7 +306,8 @@ gp_port_info_list_load (GPPortInfoList *
 	if (result < 0)
 		return (result);
 	if (list->iolib_count == 0) {
-		gp_log (GP_LOG_ERROR, "No iolibs found in '%s'", iolibs);
+		gp_log (GP_LOG_ERROR, "gphoto2-port-info-list",
+			_("No iolibs found in '%s'"), iolibs);
 		return GP_ERROR_LIBRARY;
 	}
         return (GP_OK);
